modules/util/config/TrainConfig.py

```diff
@@ -182,6 +182,7 @@ class TrainConfig(BaseConfig):
     mae_strength: float
     vb_loss_strength: float
     min_snr_gamma: float
+    dropout_pct: float
```

Comment:
I usually prefer fully descriptive variable names. `percent` instead of `pct`. But the value is between 0 and 1, so not really a percentage. Maybe `dropout_probability`?